build.sbt

```diff
@@ -280,6 +281,9 @@ lazy val simpleCodecs = alpakkaProject("simple-codecs", "simplecodecs")
 
 lazy val slick = alpakkaProject("slick", "slick", Dependencies.Slick)
 
+lazy val eventbridge =
+  alpakkaProject("aws-event-bridge", "aws.eventbridge", Dependencies.Eventbridge).disablePlugins(MimaPlugin)
```

Comment:
Have to remove this `disablePlugins` after next release in order to check the Binary Compatibiliy.